### PR TITLE
IT-2718

### PR DESCRIPTION
**Pull request by brucehoff**
IT-2718 calls for admitting external collaborators to Service Catalog.  This PR sets the Synapse team acting as the white list  to [3451917](https://www.synapse.org/#!Team:3451917), "SWNTS".

### CODE REVIEW

**Review by zaro0508 — CHANGES_REQUESTED** [review]
We occasionally need to search PRs and git history.  would you be able to help by provide a better description of this change?

**Comment by brucehoff** [comment]
> We occasionally need to search PRs and git history. would you be able to help by provide a better description of this change?

Sure, please see above.

File: config/prod/synapse-login-scipoolprod-params.yaml [hunk 1]
```diff
@@ -26,13 +26,13 @@ sceptre_user_data:
...
+    # 3451917 SWNTS (Schwannomatosis Open Research Collaborative)
     - Name: TeamToRoleArnMap
       Value: >-
         '[
           {"teamId":"3407239","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"273957","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"3409010","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"3409011","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"},
-          {"teamId":"3469335","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"}
+          {"teamId":"3451917","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"}
```
Comment (zaro0508) [inline config/prod/synapse-login-scipoolprod-params.yaml:37]: Is this intentional?  I believe the `ServiceCatalogExternalEndusers` role only has access to a limited set of products.  It does not have access to all the products that `ServiceCatalogEndusers` role does.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/synapse-login-scipoolprod-params.yaml (hunk 1 above)
Comment (brucehoff) [reply config/prod/synapse-login-scipoolprod-params.yaml:37]: Yes, it's intentional.  @allaway says, in IT-2718:

> we have a couple participants outside the US who don’t have enough local compute to explore some of the data files. We are hoping to make an rstudio instance or two available that they could use (something like a c5.xlarge or 2xlarge, nothing crazy).

So giving them access just to EC2 is sufficient.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**